simplq/src/components/pages/Admin/Token.jsx

```diff
@@ -105,7 +105,7 @@ function Token(props) {
       </div>
       <div className={styles['token-details']}>
         <div className={styles['token-name-time']}>
-          <p>Join time 11:02 pm</p>
```

Comment:
Removing joining time as it's taking up more space... but this I was not able to completely solve still... it's not very straight forward:

![image](https://user-images.githubusercontent.com/11256376/97786837-4e8fdd80-1bd4-11eb-8c00-25fd4da011b5.png)


We need to flex wrap properly or do something on small screens...